include/async/just.hpp

```diff
@@ -65,8 +65,8 @@ template <typename Tag, typename... Vs> struct sender {
         return {std::forward<R>(r), std::forward<Self>(self).values};
     }
 
-    [[nodiscard]] friend constexpr auto
-    tag_invoke(get_env_t, sender const &) noexcept -> env {
+  public:
+    [[nodiscard]] constexpr auto query(get_env_t) const noexcept -> env {
```

Comment:
question: why add `[[nodiscard]]` now?

Reply:
This did not add `[[nodiscard]]` here? But anyway, queries like this in general should be `[[nodiscard]]` and now that they are `const` member functions I think clang-tidy can flag that better.

Reply:
i see, i missed the original `[[nodiscard]]` due to the changes in formatting. 🤦🏻‍♂️